styles.css

```diff
@@ -1,9 +1,67 @@
-.title{
-  font-family:'Roboto', sans-serif;
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0; }
+
+.logo {
+  font:bold 24px/31px pt_serifbold, arial, helvetica, sans-serif;
+  padding: 40px 0;
+  padding-left: 44px;
+  position: relative; }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6195316)